lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -173,14 +173,8 @@ def safe_add_unvalidated_check_constraint(table, expression, name:)
   def unsafe_add_check_constraint(table, expression, name:, validate: true)
```

Comment:
Can we just straight up get rid of this method definition?

Reply:
For other methods, like `drop_table`, I'm not seeing an explicitly defined `unsafe_*` method where we're just delegating to `super`

Reply:
So this is different from the standard implementation I think:
```
raise ArgumentError, "Expected <name> to be present" unless name.present?
```
But it's not clear whether we want to keep it or not.

Reply:
I think I'm going to make an executive decision and say we don't need it (we don't require it for e.g. indexes).

I'll update this PR in a bit.